bench/demo.fpcore

```diff
@@ -40,3 +40,10 @@
  :name "jeff quadratic root 2"
  (let ((d (sqrt (- (* b b) (* (* 4 a) c)))))
    (if (>= b 0) (/ (* 2 c) (- (- b) d)) (/ (+ (- b) d) (* 2 a)))))
+
+(FPCore (p)
```

Comment:
Please add a `:name`. It's not always obvious what name to use, so sometimes we use names based on just syntactic characteristics of the program (like you could call it `Quotient with fmin`) or maybe this is some actual math computation that you can name. But it's good to have a memorable name.